service/history/api/updateactivityoptions/api.go

```diff
@@ -148,6 +148,7 @@ func processActivityOptionsUpdate(
 		ScheduleToStartTimeout: ai.ScheduleToStartTimeout,
 		StartToCloseTimeout:    ai.StartToCloseTimeout,
 		HeartbeatTimeout:       ai.HeartbeatTimeout,
+		Priority:               ai.Priority,
```

Comment:
hmm, this is mergeInto, so mergeActivityOptions will write directly into ai.Priority?
shouldn't we expand it like RetryPolicy? or use common.CloneProto (on both)?